datadog_lambda/tracing.py

```diff
@@ -66,6 +66,8 @@
 
         telemetry_writer.enable()
 
+is_lambda_context = os.environ.get(XrayDaemon.FUNCTION_NAME_HEADER_NAME) != ""
```

Comment:
This logic used to be in a function. But since the function name header won't change between invocations, we can parse it just once at the global level.